featuretools/tests/computational_backend/test_dask_features.py

```diff
@@ -19,12 +19,12 @@ def test_tokenize_entityset(pd_es, int_es):
     assert tokenize(pd_es) != tokenize(int_es)
 
     # add row to cohorts
-    cohorts_df = dupe['cohorts'].df
+    cohorts_df = dupe['cohorts']
     new_row = pd.DataFrame(data={'cohort': [2],
-                                 'cohort_name': ['On Time Adopters'],
+                                 'cohort_name': ['Early Adopters'],
```

Comment:
would a null value for cohort_name work here?

Reply:
Yes, a null value for `cohort_name` works.